Apps/RaptureIntTests/src/test/java/rapture/httpapi/series/SeriesApiTest.java

```diff
@@ -0,0 +1,198 @@
+package rapture.httpapi.series;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.testng.Assert;
+import org.testng.Reporter;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import rapture.common.SeriesDouble;
+import rapture.common.SeriesRepoConfig;
+import rapture.common.SeriesString;
+import rapture.common.client.HttpSeriesApi;
+import rapture.common.client.HttpLoginApi;
+import rapture.common.client.SimpleCredentialsProvider;
+import rapture.common.exception.RaptureException;
+
+public class SeriesApiTest {
+    String raptureUrl = null;
+    private String raptureUser = null;
+    private String rapturePass = null;
+    private HttpLoginApi raptureLogin = null;
+    private HttpSeriesApi seriesApi=null;
+    
+    @BeforeClass(groups={"series","cassandra", "nightly"})
+    @Parameters({"RaptureURL","RaptureUser","RapturePassword"})
+    public void beforeTest(@Optional("http://localhost:8665/rapture")String url, @Optional("rapture")String user, @Optional("rapture")String password)  {
+        raptureUrl=url;
+        raptureUser=user;
+        rapturePass=password;
+        raptureLogin = new HttpLoginApi(raptureUrl, new SimpleCredentialsProvider(raptureUser, rapturePass));
+
+        try {
+            raptureLogin.login();
+            seriesApi = new HttpSeriesApi(raptureLogin);
+
+        } catch (RaptureException e) {
+            e.printStackTrace();
+        }   
+    }
+    
+    @Test (groups={"series","cassandra", "nightly"})
+    public void testAddStringsToSeries () {
+        int MAX_VALUES=50;
+        String repoName="testSeries"+System.nanoTime();
+        String config = "SREP {} USING CASSANDRA {keyspace=\""+repoName+"KS\", cf=\""+repoName+"CF\"}";
+        seriesApi.createSeriesRepo("//"+repoName, config);
+        List <String> pointKeys = new ArrayList <String>();
+        List <String> pointValues = new ArrayList <String>();
+        for (int i = 0; i < MAX_VALUES;i++) {
+            pointKeys.add(new Integer (i).toString());
+            pointValues.add("testValue"+i);
+        }
+        String newSeries="//"+repoName+"/addStrings"+System.nanoTime();
+        Reporter.log("Adding "+pointKeys.size() + " points to "+newSeries, true);
+        seriesApi.addStringsToSeries(newSeries, pointKeys, pointValues);
+        Reporter.log("Checking points in "+newSeries, true);
+        for (SeriesString s : seriesApi.getPointsAsStrings(newSeries)) {
+            Assert.assertTrue(pointKeys.contains(s.getKey()));
+            Assert.assertTrue(pointValues.contains(s.getValue()));
```

Comment:
Aren't you doing this test the wrong way around? If getPointsAsStrings returned an empty set the test would still pass